.gitignore

```diff
@@ -15,13 +16,22 @@ config*.log
 /native/**/*.a
 /native/**/*.so
 /native/**/*.vcxproj.user
+/native/**/*.aps
 /native/**/Win32/
 /native/**/x64/
+/native/**/Release
+/native/exe/HCWIRBlasterCOM/HCWIRBlasterCOM_*
 
+/third_party/ffmpeg/lib*/*_list.c
+/third_party/ffmpeg/lib*/*.ver 
+/third_party/ffmpeg/doc 
```

Comment:
Don't add this one, it refers to an existing directory in GIT

Reply:
3 lines are listed but you said "one".  I see that "doc" should be removed but I think the other two are ok.